python/dask_cudf/dask_cudf/expr/_groupby.py

```diff
@@ -3,13 +3,55 @@
 from dask_expr._groupby import (
     GroupBy as DXGroupBy,
     SeriesGroupBy as DXSeriesGroupBy,
+    SingleAggregation,
 )
 from dask_expr._util import is_scalar
 
+from dask.dataframe.groupby import Aggregation
+
 ##
 ## Custom groupby classes
 ##
 
+
+class Collect(SingleAggregation):
+    @staticmethod
+    def groupby_chunk(*args, **kwargs):
+        return args[0].agg("collect")
```

Comment:
*suggestion*: If args is only ever allowed to be a singleton, can we either write:
```
(arg,) = args
arg.agg("collect")
```
Or (maybe better?):
```
def groupby_chunk(arg, **kwargs):
    return arg.agg("collect")
```

Similarly, if we don't handle keyword arguments does it make sense to not handle them (or at least explicitly raise if we get any).